modules/nw-bpfman-infw-configure.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * networking/network_security/bpfman-operator/bpfman-operator-about.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="bpfman-infw-configure_{context}"]
+= Configuring Ingress Node Firewall Operator to use the bpfman Operator
+
+The Ingress Node Firewall uses link:https://www.kernel.org/doc/html/latest/bpf/index.html[BPF] programs to implement some of its key firewall functionality. By default these BPF programs are loaded into the kernel using a mechanism specific to the Ingress Node Firewall.
+
+As a cluster administrator, you can configure the Ingress Node Firewall Operator to use the link:https://bpfman.io/[bpfman] Operator for loading and managing these programs instead, adding additional security and observability functionality.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have an account with administrator privileges.
+* You installed the Ingress Node Firewall Operator.
+* You have installed the bpfman Operator.
+
+.Procedure
+
+* Edit the `IngressNodeFirewallConfig` object named `ingressnodefirewallconfig` and set the `ebpfProgramManagerMode` field:
++
+.Ingress Node Firewall Operator configuration object
+[source,yaml]
+----
+apiVersion: ingressnodefirewall.openshift.io/v1alpha1
+kind: IngressNodeFirewallConfig
+metadata:
+  name: ingressnodefirewallconfig
+  namespace: ingress-node-firewall-system
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+  ebpfProgramManagerMode: <ebpf_mode>
+----
++
+--
+where:
+
+`<ebpf_mode>`: Specifies whether the Ingress Node Firewall Operator uses the bpfman Operator to manage BPF programs. Must be either `true` or `false`. If unset, bpfman is not used.
+--
```

Comment:
We need to add a list of the limitations of using bpfman with INFW somewhere. Maybe here or over in the bpfman Operator docs? Indicate that with bpfman still in Tech Preview, there are some nuances to using it. @msherif1234 has a better list, but here are a few that come to mind:
* TCX is currently not supported in bpfman, which INFW uses as a backup if XDP is not supported. So INFW won't work in this release (will be available next release) with bpfman on systems that don't support XDP (i.e. ROSA).
* When deployed with bpfman operator, the INFW daemonset will remain in the "container creating" state until the rules are applied. Everything is working fine, the INFW daemonset pods are just waiting for the eBPF maps to be created and volume mounted into the pods. It can't do anything until the maps are created anyway. This is a security feature which avoids mounting the eBPF maps on the host, which in turn avoids having to grant INFW pods access to the host filesystem.
* Ideally when INFW is deployed with bpfman operator, the deamonset pods would run as unprivileged. There is an issue with perf event arrays (still being investigated) that is preventing access without privileged mode.

Reply:
I think this is more less the complete list not sure if need to consider enforce privileged to infw as limitation probably more of new requirements